WPFHospitalEditor/SearchResultDialog.xaml.cs

```diff
@@ -338,6 +351,20 @@ private string[] EquipmentToContentRows()
             }
             return contentRows;
         }
+        private string[] MedicationToContentRows()
+        {
+            string[] contentRows = new string[HospitalMap.medicationSearchResult.Count()];
+            for (int i = 0; i < HospitalMap.medicationSearchResult.Count(); i++)
+            {
+                MapObject mo = mapObjectController.findMapObjectById(STORAGEROOM_ID);
```

Comment:
This should be out of loop because it is constant value for each iteration.

Reply:
Storage room finding is now out of the loop.